tests/library/test_parent.py

```diff
@@ -87,7 +87,7 @@ def _():
         )
     )
 
-    testdir.mkpydir("subdir").join("test_global_when.py").write(
+    pytester.mkpydir("subdir").joinpath("test_global_when.py").write_text(
```

Comment:
I think we can use the `/` operator if now it's an instance of pathlib.Path
(Not important though)